index.html

```diff
@@ -206,43 +216,46 @@ <h4>Skills</h4>
    
     <!-- FOOTER -->
     <footer class="contact">
-        <h2 class="common_title">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-        <form class="contact_form" action="https://formspree.io/f/moqraknq" method="POST">
-            <input type="text"  name="name" class="contact_input" required placeholder="Full name" maxlength="30">
-            <input type="email" name="_replyto" class="contact_input" required placeholder="Email address" >
-            <textarea name="message" class="contact_textarea"  cols="30" rows="10" required  placeholder="Enter text here" maxlength="500"></textarea>
-            <input type="submit" class="contact_button button" value="Get It Touch">
-            <ul class="home_socialbar">
-                <li class="contact_icon">
-                    <a href="#">
-                        <img src="images/icons/twitter.png" alt="twitter link">
-                    </a>
-                </li>
-                <li class="contact_icon">
-                    <a href="#">
-                        <img src="images/icons/linkedin.png" alt="linkedin link">
-                    </a>
-                </li>
-                <li class="contact_icon">
-                    <a href="#">
-                        <img src="images/icons/medium.png" alt="medium link">
-                    </a>
-                </li>
-                <li class="contact_icon">
-                    <a href="#">
-                        <img src="images/icons/github.png" alt="github link">
-                    </a>
-                </li>
-                <li class="contact_icon">
-                    <a href="#">
-                        <img src="images/icons/angellist.png" alt="angellist link">
-                    </a>
-                </li>
-                    
-                </ul>   
-            <p class="contact_email">jihane@email.com</p>
-            <img class="contact_ind" src="./images/icons/home-indicator.png" alt="home-indicator">
-        </form>
+        <div class="contact_block_1">
+            <h2 class="common_title">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <form class="contact_form" action="https://formspree.io/f/moqraknq" method="POST">
+                <input type="text"  name="name" class="contact_input" required placeholder="Full name" maxlength="30">
+                <input type="email" name="_replyto" class="contact_input" required placeholder="Email address" >
+                <textarea name="message" class="contact_textarea"  cols="30" rows="10" required  placeholder="Enter text here" maxlength="500"></textarea>
+                <input type="submit" class="contact_button button" value="Get It Touch">
+            </form>
```

Comment:
- [ ] Your design is really good 👏👏. I noticed that your button is not aligned to the right. on the Desktop if you notice that is the case. Please make this change to keep in line with the Figma 👍.
## Figma Design
![image](https://user-images.githubusercontent.com/18730106/144118525-2e74004d-eade-4836-b8e9-df109b15678c.png)

## Your Design 
![image](https://user-images.githubusercontent.com/18730106/144118405-58199dca-df2b-4bb9-9029-b4380b5b7da5.png)

- [ ] Notice also that the `textarea` on the Figma template has a green border. Please also make this change 🙏.